recipes/ace-tao/all/conanfile.py

```diff
@@ -0,0 +1,345 @@
+        # please submit PRs if you want this to build/work on other OSes
+        if self.settings.os == "Linux":
+            # If compiler is GCC, check if version is >= 4.8
+            if self.settings.compiler == "gcc":
+                self.output.info(f"compiler.version: {self.settings.compiler.version}")
+                if Version(self.settings.compiler.version) < Version("4.8"):
+                    raise ConanInvalidConfiguration(f"{self} requires GCC >= 4.8.")
+
+        if self.settings.os not in ["Linux", "Windows"]:
+            raise ConanInvalidConfiguration(
+                f"{self} is not supported on {self.settings.os}."
+            )
+
+    def build_requirements(self):
+        if self.settings.os == "Windows":
+            self.tool_requires("strawberryperl/5.32.1.1")
+
+    @property
+    def _ace_root(self):
+        return self.source_folder
+
+    @property
+    def _tao_root(self):
+        return os.path.join(self._ace_root, "TAO")
+
+    @property
+    def _msvc_toolset_version(self) -> str | None:
+        if self.settings.os != "Windows":
+            return None
+        return msvs_toolset(self)
+
+    def generate(self):
+        ace_platforms = {
+            "Linux": "linux",
+            "Windows": "windows",
+        }
+        ace_gnu_platforms = {
+            "Linux": "linux",
+            "Windows": "win32_msvc",
+        }
+
+        env = Environment()
+
+        env.define("ACE_ROOT", self._ace_root)
+        env.define("TAO_ROOT", self._tao_root)
+        env.define("INSTALL_PREFIX", self.package_folder)
+
+        if self.options.with_openssl:
+            openssl_info: CppInfo = self.dependencies.get("openssl").cpp_info
+            assert openssl_info is not None
+            env.define("SSL_ROOT", self.dependencies["openssl"].package_folder)
+
+        if self.options.with_zlib:
+            zlib_info: CppInfo = self.dependencies.get("zlib").cpp_info
+            assert zlib_info is not None
+            env.define("ZLIB_ROOT", self.dependencies["zlib"].package_folder)
+
+        envvars = env.vars(self)
+        self.output.info(f"my_env_file will contain: {envvars.items()}")
+        envvars.save_script("my_env_file")
+
+        # Create a config file
+        ace_platform = ace_platforms[str(self.settings.os)]
+        ace_gnu_platform = ace_gnu_platforms[str(self.settings.os)]
+        config_path = os.path.join(self._ace_root, "ace", "config.h")
+        save(self, config_path, f'#include "ace/config-{ace_platform}.h"\n')
+
+        # Create the makefile parameters
+        build_conf_path = os.path.join(
+            self._ace_root, "include", "makeinclude", "platform_macros.GNU"
+        )
+        # see wrapper_macros.GNU for all options
+
+        debug = bool(self.options.debug) or (self.settings.build_type == "Debug")
+        build_config = (
+            f"shared_lib={_BOOL_STR[bool(self.options.shared)]}\n"
```

Comment:
```suggestion
            f"shared_libs={_BOOL_STR[bool(self.options.shared)]}\n"
```
`static_libs` => `shared_libs`

Reply:
Good catch, thanks a lot!